casanovo/config.yaml

```diff
@@ -65,6 +65,7 @@ weight_decay: 1e-5
 # Training/inference options.
 train_batch_size: 32
 predict_batch_size: 1024
+n_beams: 1
```

Comment:
`n_beams` determines both the number of beams explored and total number of beams cached. Model currently only reports a single PSM regardless of this parameter. We can add functionality to report multiple PSMs later.

I implemented beams explored = beams cached both because it's straight-forward but also because that's the minimum number of peptide predictions we are guaranteed to come up with for a given number of beams, i.e. larger cache than number of beamse doesn't guarantee more PSMs (if/when we start reporting multiple PSMs)

Reply:
Beams explored = beams cached is fine I think, but we will want an option (as a future feature addition is fine) to report multiple PSMs per spectrum.

When considering only a single beam, the result is the same as for greedy decoding. So we probably want to increase the default number of beams a bit to actually get advantage of this functionality.

Reply:
Raising it to 5 and opening an issue for multi-PSM output for future reference